docs/reference/announcements.md

```diff
@@ -10,6 +10,34 @@ Scheduled release date: 8th of Oct 2024
 
 Scheduled end of maintenance: 14th of April 2026
 
+### Configuring license keys
```

Comment:
I propose to keep the announcement short and focus on:
1. With 8.6 Camunda requires a license key to run in production (link to blog post)
2. Please find out here how to set the license key (link to install instructions for sm)

Reply:
Adjusted in latest commit 👍